### PR TITLE
WIP: Help replacement

### PR DESCRIPTION
**Pull request by jburel**
Replace links to help
Work in progress
Some sections of the web pages need to be discussed and changed

cc @pwalczysko @will-moore @joshmoore @sbesson

### CODE REVIEW

**Comment by pwalczysko** [comment]
failed checks

File: _posts/2014-02-25-omero-bio-formats-5-0-0.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ For OMERO:
 - CLI plugin allowing access to tags and tag sets from the [command line](https://www.openmicroscopy.org/site/support/omero5.0/users/command-line-interface.html)
 - Modulo (FLIM/SPIM) support in OMERO.insight
 - Many bug fixes including thumbnails, plates, tables and LDAP
-- New help guide for installing the plugins needed to use [OMERO with ImageJ](https://help.openmicroscopy.org/imagej.html)
+- New help guide for installing the plugins needed to use [OMERO with ImageJ](https://omero-guides.readthedocs.io/en/latest/fiji/docs/index.html)
```
Comment (sbesson) [inline _posts/2014-02-25-omero-bio-formats-5-0-0.md:67]: Minor comment: i am not sure we want to edit historical timestamped announcement posts esp. when we control these URLs and plan to remap them.

Comment (jburel) [reply _posts/2014-02-25-omero-bio-formats-5-0-0.md:67]: we will redirecting. I updated them since there were only a few to adjust

Comment (jburel) [reply _posts/2014-02-25-omero-bio-formats-5-0-0.md]: removed commit from the git history

File: explore/index.html [hunk 2]
```diff
@@ -24,7 +24,7 @@ <h2 class="header-red">OME for Scientists</h2>
         <div class="row">
             <img class="medium-3 medium-offset-1 columns" alt="OMERO for Scientists" src="{{ site.baseurl }}/img/icons/user-groups_scientists.svg">
             <div class="medium-8 columns">
-                <p>Sign up for our <a href="https://help.openmicroscopy.org/demo-server.html">demo server</a> to test out the functionality with your own data.</p>
+                <p>Sign up for our demo server to test out the functionality with your own data.</p>
```
Comment (pwalczysko) [inline explore/index.html:27]: But now there is no link at all to go to.

I think in the meantime, we should point to http://qa.openmicroscopy.org.uk/registry/demo_account/

**Comment by jburel** [comment]
Closing this PR for now to avoid conflict with #395

**Pull request closed without merging**